gatsby-config.js

```diff
@@ -19,30 +19,6 @@ module.exports =  {
           icon: "src/images/logo.svg",
         },
       },
```

Comment:
we can probably remove the [gatsby-plugin-image](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/?=gatsby-plugin-image) as we don't use it and it was using gatsby-plugin-sharp and  gatsby-transformer-sharp that have been removed

Reply:
done in b526efd